tests/unit/objectives/test_single_objectives.py

```diff
@@ -37,6 +37,13 @@
     LOGARITHMIC_GOLDSTEIN_PRICE_MINIMIZER,
     LOGARITHMIC_GOLDSTEIN_PRICE_MINIMUM,
     LOGARITHMIC_GOLDSTEIN_PRICE_SEARCH_SPACE,
+    MICHALEWICZ_2_MINIMIZER,
+    MICHALEWICZ_2_MINIMUM,
+    MICHALEWICZ_2_SEARCH_SPACE,
+    MICHALEWICZ_5_MINIMUM,
+    MICHALEWICZ_5_SEARCH_SPACE,
+    MICHALEWICZ_10_MINIMUM,
+    MICHALEWICZ_10_SEARCH_SPACE,
```

Comment:
Where are the minimizers for 2 and 5?

Reply:
I don't know them, they're not listed on the [https://www.sfu.ca/~ssurjano/michal.html](url) site...

Reply:
Work them out? Probably useful to have them?